README.md

```diff
@@ -73,6 +75,45 @@ mysensor.off()
 
 ```
 
+### Switch
+
+The switch is similar to a _binary sensor_, but in addition to publishing state changes toward HA it can also receive 'commands' from HA that request a state change.
+It is possible to act upon reception of this 'command', by defining a `callback` function, as the following example shows:
+
+#### Usage
+
+```py
+from ha_mqtt_discoverable import Settings
+from ha_mqtt_discoverable.sensors import Switch, SwitchInfo
+from paho.mqtt.client import Client, MQTTMessage
+
+# Configure the required parameters for the MQTT broker
+mqtt_settings = Settings.MQTT(host="localhost")
+
+# Information about the switch
+# If `command_topic` is defined, it will receive state updates from HA
+switch_info = SwitchInfo(name="test", command_topic="command")
+
+settings = Settings(mqtt=mqtt_settings, entity=switch_info)
+
+# To receive state commands from HA, define a callback function:
+def my_callback(client: Client, user_data, message: MQTTMessage):
```

Comment:
I can already think of some good uses for this. Appreciate how well you've been documenting the example code in your PRs as you add functionality.